From 8b0acb452cf0a1bab81b5c3a90e088f9e8c57068 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Mon, 6 Aug 2018 20:08:10 +0000
Subject: [PATCH] TRITON-668 Add SRV records to FWAPI Reviewed by: Nick
 Zivkovic <nick.zivkovic@joyent.com> Approved by: Nick Zivkovic
 <nick.zivkovic@joyent.com>

---
 sapi_manifests/registrar/manifest.json |  5 +++++
 sapi_manifests/registrar/template      | 30 ++++++++++++++++++++++++++
 2 files changed, 35 insertions(+)
 create mode 100644 sapi_manifests/registrar/manifest.json
 create mode 100644 sapi_manifests/registrar/template

diff --git a/sapi_manifests/registrar/manifest.json b/sapi_manifests/registrar/manifest.json
new file mode 100644
index 0000000..98cd3bb
--- /dev/null
+++ b/sapi_manifests/registrar/manifest.json
@@ -0,0 +1,5 @@
+{
+    "name": "registrar",
+    "path": "/opt/smartdc/registrar/etc/config.json",
+    "master": true
+}
diff --git a/sapi_manifests/registrar/template b/sapi_manifests/registrar/template
new file mode 100644
index 0000000..8236209
--- /dev/null
+++ b/sapi_manifests/registrar/template
@@ -0,0 +1,30 @@
+{
+  "registration": {
+    "domain": "{{SERVICE_DOMAIN}}",
+    "type": "rr_host",
+    "service": {
+      "type": "service",
+      "service": {
+        "srvce": "_http",
+        "proto": "_tcp",
+        "port": 80
+      },
+      "ttl": 60
+    },
+    "ttl": 60
+  },
+
+  "zookeeper": {
+    "servers": [
+      {{#ZK_SERVERS}}
+        {
+          "host": "{{host}}",
+          "port": {{port}}
+        }{{^last}}, {{/last}}
+      {{/ZK_SERVERS}}
+    ],
+    "timeout": 60000
+  },
+
+  "adminIp": "{{auto.ADMIN_IP}}"
+}
-- 
2.21.0

